index.html

```diff
@@ -2434,6 +2434,59 @@ <h3>
           </p>
         </aside>
       </section>
+      <section class="informative">
+        <h3>
+          Web Apps with scope conflicts
+        </h3>
+        <p>
+          It is possible for two web apps to be installed where there are
+          conflicts between the scope. These conflicts come with disadvantages
+          outlined below and they are generally not recommended.
+        </p>
+        <ul>
+          <li>The scopes of the two web apps can be on the same origin. Not
+          recommended.
+          </li>
+          <li>The scope of one web app can be nested inside the scope of the
+          other. Strongly not recommended.
+          </li>
+          <li>The scopes of the two web apps can be the same. Strongly not
+          recommended.
+          </li>
+        </ul>
+        <p>
+          Same-origin scopes are not recommended due to origin-based settings
+          that will affect all apps installed under that origin. Settings like:
+        </p>
+        <ul>
+          <li>Permissions
+          </li>
+          <li>Storage and storage quota
+          </li>
+          <li>User settings (e.g. font size)
+          </li>
+        </ul>
+        <p>
+          Nested scopes are strongly not recommended. They have the above
+          complications of same-origin scope, along with the following UX and
+          API problems or inconsistencies among other possible consistencies:
+        </p>
+        <ul>
```

Comment:
I think this is getting into user agent behaviour... and that might be a good thing for us to specify.

The simplest model might be last installed and/or most specific wins.

Reply:
As mentioned in another comment - I'm playing around with adding a definition here, but perhaps we want to pull out a section per API that is affected, and then reference the "controlling app" concept in that section? What do you think?

Reply:
+1 for specifying the behaviour around this (perhaps starting with MAY requirements so we don't require browsers to update right away). It doesn't feel good to say "not work" as a possible outcome.

For simplicity, I'm OK with leaving this here for now, but long-term as you say it would be good for each section to specify "Choose the controlling app".

There's actually prior art for this which we might want to merge into this concept: in the Badging API there's "[determining the set of matching applications](https://w3c.github.io/badging/#determining-the-set-of-matching-applications)" which is a bit weird in that it returns a set but usually only with one item in it. If you just look at the document case (the top half), it's explicitly picking the one with the most specific scope.